ci/docker/Dockerfile.alpine-release

```diff
@@ -0,0 +1,51 @@
+FROM alpine:3.15 as build
+
+# creates /opensmtpd dir and makes all following commands to run in it
+# https://docs.docker.com/engine/reference/builder/#workdir
+WORKDIR /opensmtpd
+
+# install necessary packages
+RUN apk add --no-cache \
+    autoconf \
+    automake \
+    bison \
+    ca-certificates \
+    fts-dev \
+    gcc \
+    fts \
+    libevent-dev \
+    libtool \
+    libtool \
```

Comment:
This is an issue with the current Dockerfile as well, but there's no need to specify `libtool` twice.  Actually, the exact list of dependencies should be re-evaluated probably, but that's a separate issue from this PR.

Reply:
Oh, I wasn't focused on the build part. I'll make this fix in both dockerfiles if thats okay

Reply:
Not a big deal.  It's a separate issue, so it's fine with me if you prefer to fix afterwards, and regarding the CI dockerfiles they need some small tweaks anyway.  Oh, patches and are always welcome though :)

Reply:
Fixed in the dockerfile im adding.